src/components/views/Instance/Edit/Repository.tsx

```diff
@@ -117,6 +117,7 @@ export default function Repository(): JSX.Element {
                 .then(prs => {
                     if (prs.code === StatusCode.ERROR) {
                         addError(errorState, prs.error);
+                        setPRs(PRs ?? []);
                     } else {
                         setPRs(prs.payload);
```

Comment:
Problem is you still cant remove any prs with this

Reply:
Wouldn't your change fix that?

Reply:
Negative, the PR list cannot be displayed without info from github and without a pr list, you can see or interact with merged PRs

Reply:
> the PR list cannot be displayed without info from github

We really need to change that so it can work with the existing test merge list